src/Admin/Pool.php

```diff
@@ -259,6 +267,19 @@ public function getAdminByAdminCode($adminCode)
             }
 
             if (!$admin->hasChild($code)) {
+                @trigger_error(sprintf(
+                    'Passing an invalid admin hierarchy inside argument 1 for %s() is deprecated since 3.x and will throw an exception in 4.0.',
```

Comment:
```suggestion
                    'Passing an invalid admin hierarchy inside argument 1 for %s() is deprecated since sonata-project/admin-bundle 3.x and will throw an exception in 4.0.',
```